File: CmdMents/TimesAndSeasons/AfflictYourselfOnYomKippur.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CmdMents.TimesAndSeasons
{
    class AfflictYourselfOnYomKippur : NoWorkingOnYomKippur
    {
        public AfflictYourselfOnYomKippur()
        {
            base.AlternateText = "This is to be a lasting ordinance for you: On the tenth day of the seventh month you must deny yourselves and fast.";
            base.Book = CommandmentBook.Leviticus;
            base.CanBeCarriedOutToday = true;
            base.Chapter = 16;
            base.CommandmentType = CommandmentType.Positive;
            base.FollowedByChristians = false;
            base.FollowedByMessianics = true;
            base.FollowedByObservantJews = true;
            base.Number = 94;
            base.ShortSummary = "Afflict yourselves on Yom Kippur.";
            base.Text = "This is to be a lasting ordinance for you: On the tenth day of the seventh month you must deny yourselves.";
            base.Verse = 29;
        }
    }
}

File: EtzMitzvot/Commandments/TimesAndSeasons/afflictYourselfOnYomKippur.ts
///<reference path='../commandmentBase.ts' />

class AfflictYourselfOnYomKippur extends CommandmentBase {
    constructor() {
        super();

        this.shortSummary = "Afflict yourself on Yom Kippur";
        this.text = "This is to be a lasting ordinance for you: On the tenth day of the seventh month you must deny yourselves and fast.";
        this.book = TorahBook.Leviticus;
        this.chapter = 16;
        this.verse = 29;

        this.canBeCarriedOutToday = true;
        this.requiresLivingInIsrael = false;

        this.type = CommandmentType.Positive;
        this.christianObservance = CommandmentObedience.None;
        this.messianicObservance = CommandmentObedience.Binding;
        this.jewishObservance = CommandmentObedience.Binding;
        this.number = 94;

        this.children = [
            new FastOnYomKippur()
        ];
    }
} 